mis_builder/static/src/components/mis_report_widget.xml

```diff
@@ -13,12 +13,11 @@
                         </div>
                         <div class="oe_mis_builder_cp_right_bottom">
                             <div class="oe_mis_builder_filter_buttons">
-                                <FilterMenu t-if="showSearchBar" />
-                                <DatePicker
-                                    date="state.pivot_date"
-                                    onDateTimeChanged="onDateTimeChanged.bind(this)"
+                                <DateTimeInput
                                     placeholder="'Base date...'"
+                                    type="'date'"
                                     t-if="showPivotDate"
+                                    onChange="onDateTimeChanged.bind(this)"
```

Comment:
Does this some sort of binding to `state.pivot_date`?

Reply:
Yes, it works exactly as v16. The called function is this one:
`onDateTimeChanged(ev) {
        this.state.pivot_date = ev;
        this.refresh();
    }`

Reply:
There is something missing, because in 16, you could set the Base Date on the report form, and it was preserved when clicking Preview. 

![image](https://github.com/OCA/mis-builder/assets/692075/3a050063-887a-494d-aea9-60604242b92b)

![image](https://github.com/OCA/mis-builder/assets/692075/0df2cb37-8109-4c0f-8414-7b5779f82c94)

![image](https://github.com/OCA/mis-builder/assets/692075/aa97f20c-2083-4662-ac32-5230c13f4233)

With this branch, it does not appear in preview. You can check on [runboat](http://oca-mis-builder-17-0-pr607-17b3f9776239.runboat.odoo-community.org/web#id=1&cids=1&menu_id=184&action=292&model=mis.report.instance&view_type=form).

Reply:
Fixed. In v16 the DatePicker date attribute was setted with state.pivot_date value. Since DateTimeInput doesnt have this attribute i setted it in value.